distributed/_version.py

```diff
@@ -84,7 +84,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
                 stderr=(subprocess.PIPE if hide_stderr else None),
```

Comment:
This file is auto-generated by versioneer and IMHO should be left untouched and excluded from pyupgrade

Reply:
I'll revert. FWIW we already formatted the file in a black formatting step ~2years ago